modules/ROOT/pages/stats-monitoring.adoc

```diff
@@ -196,6 +196,26 @@ This metric can be used to calculate the time between seeing a change on the DCP
 
 DCP cache latency = `dcp_caching_time` / `dcp_caching_count`
 
+===== num_pull_repl_active_continuous
+
+Gauge representing the number of active replications, be it continuous or one shot.
+This stat is identical to `num_pull_repl_active_one_shot`.
```

Comment:
@adamcfraser @bbrks I need a confirmation for this stat. From [row 77 and 78](https://docs.google.com/spreadsheets/d/1aZjCIwwZYrUSaQCK63QUg7_jrbo0jJcsxPoXG2Q4s0o/edit#gid=0):

> I don't believe SG can distinguish between one-shot and continuous push replications (that's client-driven).

Can you confirm?

Reply:
@jamiltz For **_pull_** replications, we can distinguish between the two, so we track each replication type individually (note: all of these stats are pull-replication specific)

Reply:
Updated in https://github.com/couchbase/docs-sync-gateway/pull/68/commits/0c1fb46d44baa2c5abb78526850d7361b206f453